### PR TITLE
codebraid: 0.5.0 -> 0.11.0

### LINKED ISSUE #257670: Update request: codebraid 0.5.0-unstable-2020-08-14 → 0.10.3
- Package name: codebraid
- Latest released version: [0.10.3](https://pypi.org/project/codebraid/0.10.3/)

- Current version on the unstable channel: 0.5.0-unstable-2020-08-14
- Current version on the stable/release channel: 0.5.0-unstable-2020-08-14

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

@Synthetica9 

The currently packaged version of codebraid does not work with the currently packaged version of pandoc as it's too new. The (mathematically weird) error message is `RuntimeError: Pandoc at "pandoc" is version 3.0.1.0, but >= 2.4 is required`... 😆  If I get time before someone else does I'll make a PR, but flagging it up anyway.

-----

Note for maintainers: Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by chewblacka**
## Description of changes
https://github.com/gpoore/codebraid/releases/tag/v0.11.0
Closes #257670

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by FliegendeWurst — CHANGES_REQUESTED** [review]
v0.11.0 was released recently.

**Comment by chewblacka** [comment]
> v0.11.0 was released recently.

v0.11.0, needs python3Packages.bespon 0.7.0 (currently at 0.6.0).
I've created a PR for bespon here:
https://github.com/NixOS/nixpkgs/pull/261822

**Comment by chewblacka** [comment]
> v0.11.0 was released recently.

As soon as someone approves #261822 for merger I'll bump this up to v0.11.0

File: pkgs/tools/misc/codebraid/default.nix [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "codebraid";
-  version = "0.5.0-unstable-2020-08-14";
+  version = "0.11.0";
```
Comment (NickCao) [inline pkgs/tools/misc/codebraid/default.nix:5]:
```suggestion
  version = "0.11.0";
  pyproject = true;
```

File: pkgs/tools/misc/codebraid/default.nix [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
     owner = "gpoore";
     repo = pname;
-    rev = "526a223c4fc32c37d6c5c9133524dfa0e1811ca4";
-    sha256 = "0qkqaj49k584qzgx9jlsf5vlv4lq7x403s1kig8v87i0kgh55p56";
+    rev = "v${version}";
+    hash = "sha256-E9vzGK9ZEVwF+UBpSkdM+hm6vINen/A+LgnnPpc77QQ=";
   };
 
-  propagatedBuildInputs = with python3Packages; [ bespon ];
+  propagatedBuildInputs = with python3Packages; [ bespon setuptools ];
```
Comment (NickCao) [inline pkgs/tools/misc/codebraid/default.nix:14]: `setuptools` should be in `nativeBuildInputs`

Comment (chewblacka) [reply pkgs/tools/misc/codebraid/default.nix]: Many thanks for those fixes. Much appreciated.

**Review by NickCao — APPROVED** [review]

**Pull request merged**